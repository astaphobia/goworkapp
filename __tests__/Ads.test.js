import React from 'react'
import { shallow } from 'enzyme'

import Ads from '@/components/Ads'

describe('Ads', () => {
    it('hope doing well on Composing component', () => {
        const comp = shallow(<Ads />)
        expect(comp).toMatchSnapshot()
    })
})