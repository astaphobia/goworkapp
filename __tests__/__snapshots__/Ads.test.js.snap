// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ads hope doing well on Composing component 1`] = `<Component />`;
